samples/cpp/videocapture_audio_combination.cpp

```diff
@@ -0,0 +1,56 @@
+#include <opencv2/core.hpp>
+#include <opencv2/videoio.hpp>
+#include <opencv2/highgui.hpp>
+#include <iostream>
+#include <stdio.h>
+#include <thread>
+
+using namespace cv;
+using namespace std;
+
+int main(int, char**)
+{
+    Mat frame_video;
+    Mat frame_audio;
+    std::vector<Mat> video_data;
+    Mat audio_data;
+
+    int apiID = cv::CAP_MSMF;
+    //congigurate VideoCapture for video and audio
+    VideoCapture cap_video;
+    VideoCapture cap_audio;
```

Comment:
Wrong way.

As said above, we are integrating into VideoCapture to handle both streams instead of adding a dedicated Audio API.